creator-node/src/components/healthCheck/healthCheckComponentService.js

```diff
@@ -50,14 +50,16 @@ const healthCheck = async ({ libs } = {}, logger, sequelize, randomBytesToSign =
  * @param {*} ServiceRegistry
  * @param {*} logger
  */
-const healthCheckVerbose = async ({ libs } = {}, logger, sequelize, getMonitors) => {
+const healthCheckVerbose = async ({ libs } = {}, logger, sequelize, getMonitors, os) => {
```

Comment:
you pass in os here vs importing so you can mock it in the tests?

Reply:
that's correct. this flow is also similar to that of the `getMonitors` logic, where the actual logic is also passed in